Assets/__Scripts/Weapon.cs

```diff
@@ -31,10 +31,10 @@ public class WeaponDefinition
     public Color color = Color.white; // Color of Collar & power-up
     public GameObject projectilePrefab; // Prefab for projectiles
     public Color projectileColor = Color.white;
-    public float damageOnHit = 0; // Amount of damage caused
+    public float damageOnHit = -1; // Amount of damage caused
```

Comment:
These are meant to be set in the MainScript in the WeaponDef array so that each weapon can use different values. These here are just the defaults